### PR TITLE
workaround succeed re.booleans in UCR

### PR DESCRIPTION
**Pull request by sravfeyn**
http://manage.dimagi.com/default.asp?178217#989590

### CODE REVIEW

**Comment by sravfeyn** [comment]
@esoergel

**Comment by sravfeyn** [comment]
fyi @czue

**Comment by czue** [comment]
thanks! https://github.com/dimagi/commcare-hq/pull/7783/

File: custom/succeed/data_sources/patient_task_list.json [hunk 1]
```diff
@@ -78,7 +78,7 @@
                             "datatype": "string"
                         },
                         "type": "boolean_expression",
-                        "property_value": "true"
+                        "property_value": "True"
```
Comment (esoergel) [inline custom/succeed/data_sources/patient_task_list.json]: Shouldn't it be `true` (without the quotes)?  JSON supports first-class boolean data types.  Or does this need to be a string for some other reason?

Comment (esoergel) [inline custom/succeed/data_sources/patient_task_list.json]: Nevermind - just saw Cory's link to https://github.com/dimagi/commcare-hq/pull/7783

**Pull request merged**